tdrs-backend/tdpservice/parsers/test/factories.py

```diff
@@ -1,7 +1,10 @@
 """Factories for generating test data for parsers."""
```

Comment:
The T2 record's fields should almost all be alphanumeric but they have been changed to strictly numerics. Need to convert these all back to alphanumerics and update the validators appropriately. T1 and T3 should be revalidated as well to make sure their field types have not regressed.

Reply:
This comment is erroneous since we arent following the pdfs anymore and we are following the spreadsheets.